floodviz/static/js/tooltip.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+const tooltip_module = function (options) {
```

Comment:
This isn't really using the module pattern. This is really just a single stateless function show_tooltip. I would be okay just exposing the show_tooltip function. Maybe something like FV.show_tooltip